sdk/tables/azure-data-tables/azure/data/tables/_table_client.py

```diff
@@ -34,9 +34,19 @@
 class TableClient(TablesBaseClient):
     """A client to interact with a specific Table in an Azure Tables account.
 
-    :ivar str account_name: The name of the Tables account.
-    :ivar str table_name: The name of the table.
-    :ivar str url: The full URL to the Tables account.
+    :param str endpoint: A URL to an Azure Tables account.
+    :param str table_name: The name of the table.
+    :keyword credential:
+        The credentials with which to authenticate. This is optional if the
+        account URL already has a SAS token. The value can be one of AzureNamedKeyCredential,
+        AzureSasCredential, or TokenCredential from azure-core.
+    :paramtype credential:
+        ~azure.core.credentials.AzureNamedKeyCredential or
+        ~azure.core.credentials.AzureSasCredential or
+        ~azure.core.credentials.TokenCredential or None
+    :keyword api_version: Specifies the version of the operation to use for this request. Default value
```

Comment:
Lets remove "Note that overriding this default value may result in unsupported behavior." :)